### ISSUE TITLE
Hello, i have imported this project, i have established physical connection but not listening to any audio call

### ISSUE THREAD

**Issue by raeeskhan3370** [danInAustralia/WalkieTalkie#6]

**Comment by danInAustralia** [comment]
Hi @raeeskhan3370 . Thanks for your comment. I'll look into this. It was working with audio on a basic level. My attention has been on other projects recently. Just out of interest, did you install this on two devices? The code doesn't yet check that the service is running on the other wifi direct device.

**Comment by raeeskhan3370** [comment]
Yes i have installed on two devices,but its not working,can you fix that
because i need this urgently for my product.
